### PR TITLE
Address all FIgma comments and UI text changes in Figma files for login, join, waitlist and add logic for waitlist env variable

### PR DESCRIPTION
**Pull request by Excellent-coding**
https://linear.app/find-labs/issue/ENG-26/address-all-figma-comments-and-ui-text-changes-in-figma-files-for

- Connect Payment
- Header, Footer, Auth links
- Text and UI update

### CODE REVIEW

File: apps/web-app/src/components/CheckoutForm.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+const CheckoutForm = () => {
+  const stripe = useStripe();
+  const elements = useElements();
+
+  const [message, setMessage] = useState<string>();
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleSubmit = async (e: any) => {
+    e.preventDefault();
```
Comment (khalil-omer) [inline apps/web-app/src/components/CheckoutForm.tsx:22]: @Excellent-coding don't we need to handle submit?

File: apps/web-app/src/pages/connect-payment.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import Image from "next/image"
+import LockIcon from "../assets/icon/lock.svg"
+import { loadStripe } from "@stripe/stripe-js";
+import { Elements } from "@stripe/react-stripe-js";
+import CheckoutForm from "../components/CheckoutForm"
+
+const stripePromise = loadStripe('pk_live_Qr3xcCMK3oQ9rs1GRDmsxVyf004BexykCf');
```
Comment (khalil-omer) [inline apps/web-app/src/pages/connect-payment.tsx:7]: This key should load from an env.

File: apps/web-app/src/pages/connect-payment.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      for $5 a month
+    </div>
+    <p className="text-gray-400 text-sm mb-12 font-semibold max-w-lg mx-auto">
+      Your Find subscription will renew automatically every month for $5. You can cancel your subscription any time.
+    </p>
+    <div className="grid lg:grid-cols-2 space gap-6 text-left">
+      <div>
+        <h2 className="font-semibold text-lg mb-3">What you get</h2>
+        <ul className="bg-gray-100 p-5 rounded list-disc">
+          <li className="flex my-3 text-gray-500"><div className="w-4 h-4 rounded-full bg-gray-400 my-auto mr-2" />Sed posuere liqula a nulla sagittis, et consectetur.</li>
```
Comment (khalil-omer) [inline apps/web-app/src/pages/connect-payment.tsx:24]: @Excellent-coding  I think you are using the _wireframe_ tab in Figma. Don't use that tab. Use the **desktop** tab which has all of the correct text.

**Review by khalil-omer — CHANGES_REQUESTED** [review]

File: apps/web-app/src/components/CheckoutForm.tsx (hunk 1 above)
Comment (Excellent-coding) [reply apps/web-app/src/components/CheckoutForm.tsx:22]: Now this is only design for now
I will integrate stripe at ENG-30 card

File: apps/web-app/src/pages/waitlist.tsx [hunk 4]
```diff
@@ -30,34 +31,40 @@ const JoinWaitlist = () => {
...
             value={email}
-            onChange={e => setEmail(e.target.value)}
-            className="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+            onChange={setEmail}
           />
-          <div className="text-red-500 text-sm font-medium">
-          </div>
         </div>
       </div>
       <Button
```
Comment (khalil-omer) [inline apps/web-app/src/pages/waitlist.tsx:70]: Button color wrong

File: apps/web-app/src/pages/waitlist.tsx [hunk 5]
```diff
@@ -30,34 +31,40 @@ const JoinWaitlist = () => {
...
       </div>
-      <div className="flex flex-wrap mb-3">
+      <div className="flex flex-wrap mb-3 mt-8">
         <div className="w-full text-left">
-          <label className="block tracking-wide text-gray-700 text-xs font-bold mb-2">
-            Email
-          </label>
-          <input
-            type="email"
+          <Input
```
Comment (khalil-omer) [inline apps/web-app/src/pages/waitlist.tsx:62]: Input bg color wrong.

**Review by khalil-omer — CHANGES_REQUESTED** [review]
Don't use the wireframe tab of the Figma file. Use 'UI Desktop'

File: apps/web-app/src/pages/waitlist.tsx (hunk 5 above)
Comment (Excellent-coding) [reply apps/web-app/src/pages/waitlist.tsx:62]: I think now dark mode has same colors, so I will work on dark-mode branch(eng-32) and will create a PR for this ticket
Let's close this PR, I will create new PR based on the ENG-32 branch

**Review by khalil-omer — CHANGES_REQUESTED** [review]
Conflicts

**Pull request closed without merging**